### ISSUE TITLE
New issue

### ISSUE THREAD

**Issue by Riplinian** [Mirdain/Silmaril#2]
![Silmaril issue](https://user-images.githubusercontent.com/49185395/224530882-55249661-7bdb-428c-962e-2caaa2e9baaa.png)
Tried the new version, got this message.

**Comment by Mirdain** [comment]
What version is this and did you replace and reload the addon (lua files)?  I added the luas .zip to the 1.3.1 (same lua files as 1.3)

**Comment by Riplinian** [comment]
I tried the 1.3, the 1.3.1, all with the new luas, and i tried it with the old. I eneded up going back to 1.2, but then I encountered a follow issue.

**Comment by Riplinian** [comment]
> 

I'll try again. Maybe its me. Trying from scratch later. Thnx 😀

**Comment by Riplinian** [comment]
Still that same issue.

**Comment by Mirdain** [comment]
Did you unload ALL characters by a //lua u silmaril the a //lua l silmaril ?

**Comment by Riplinian** [comment]
Yea. Unloaded silmaril and gearswap. Deleted silmaril. Downloaded the 1.3.1 again. Unblocked, unpacked, put in addons.  Loaded silmaril,  started .exe, got the same message. Unloaded everythn, deleted everythn again. Redid with 1.2, same steps, works.

**Comment by Mirdain** [comment]
What language is your pc using?

**Comment by Riplinian** [comment]
Its in Norwegian.

**Comment by Riplinian** [comment]
But that shouldnt matter, if the 1.1 and the 1.2 works. Right?

**Comment by Mirdain** [comment]
I changed the lang handling and removed cultures to try to help.  Seems it
might have broken it for you.  It was 1.3 change.

**Comment by Riplinian** [comment]
Ah. If that makes you program better, thats cool. Aslong as I can use 1.2, i'm a happy camper. Silmaril is so good 😀

**Comment by Mirdain** [comment]
Ironically I was trying to get it to work for a friend in Norweigan...  I
will try and trouble shoot some more for next release.  He swapped his pc
to English for now.....

**Comment by Riplinian** [comment]
I'm not a computer guy really. But changing my computer language to english, that would make it work?

**Comment by Mirdain** [comment]
Should

**Comment by Mirdain** [comment]
Fixed by forcing the lua to English and the Addon thread in Silmaril.
